### DISCUSSION TITLE
Cannot run Milvus in our Openshift cluster with (runsAsUser, runAsGroup, fsGroup) - https://github.com/zilliztech/milvus-helm/issues/43

### DISCUSSION THREAD

**Discussion by tmechineni12** [milvus-io/milvus#29375] [Ideas & Feature requests]
Hi All, 

Can anyone please see the issue I submitted today and explain how to fix this ? or let me know if you ran into similar problem before ?? 

https://github.com/zilliztech/milvus-helm/issues/43

Thanks!
Tharun

**Comment by yhmo** [comment]
Wait for the milvus-helm owner's comment in the issue.
